bleachbit/Special.py

```diff
@@ -41,12 +41,12 @@ def __shred_sqlite_char_columns(table, cols=None, where=""):
     """Create an SQL command to shred character columns"""
     cmd = ""
     if cols and options.get('shred'):
+        #cmd += "update %s set %s %s;" % \
```

Comment:
I would like to address the SQL error in another way, so would you please remove it from this pull request?